src/inventory_client/inventory_client.go

```diff
@@ -80,6 +81,11 @@ func CreateInventoryClient(clusterId string, inventoryURL string, pullSecret str
 			RootCAs:            certs,
 		},
 	})
+	// Add retry settings
+	clientConfig.Transport = rehttp.NewTransport(transport,
```

Comment:
nice

Reply:
rehttp doesn't seem to work.:leftwards_arrow_with_hook: 
Wrote a simple retry roundtripper myself